pom.xml

```diff
@@ -37,7 +37,7 @@
     <connection>scm:git:git://github.com/stapler/stapler.git</connection>
     <developerConnection>scm:git:ssh://git@github.com/stapler/stapler.git</developerConnection>
     <url>https://github.com/stapler/stapler</url>
-    <tag>HEAD</tag>
+    <tag>${scmTag}</tag>
```

Comment:
Does this work fine with creating releases?

Reply:
Apparently it does. Here's the documentation on it: [Incrementals](https://www.google.com/url?q=https://rmh.psdschools.org/webfm/1875&sa=U&ved=0ahUKEwjpxcvIo9fgAhVFYK0KHdbyAN0QFggEMAA&client=internal-uds-cse&cx=004191848107721260813:sww1jghnesa&usg=AOvVaw0zpO40o24JrwsU9n9fkD1t)

And the changes the automated incrementalization tool added in Remoting for reference: [Add incremtentals to Remoting](https://github.com/jenkinsci/remoting/commit/33bc3f99f7155ec090e01bbd158cd3105cd2e487). I've successfully released Remoting after that.

Reply:
scmTag is defined in the parent pom typically, and the release stuff sets that automatically (from incrementals).